### PR TITLE
[BEHAT]added new query type & added support for new query type in ContentTyp…

### PR DESCRIPTION
**Pull request by tomaszszopinski**
JIRA: https://jira.ez.no/browse/EZEE-3237
Added support for FoldersUnderMediaQueryType in. services.yaml
Added support for usage of Parameters & ItemsPerPage in creation of content type.
Created new query & readme file in src/lib/QueryType/ folder.

### CODE REVIEW

**Comment by mnocon** [comment]
Travis is failing:
1) CS issues
2) https://github.com/ezsystems/BehatBundle/blob/8.1/features/examples/content.feature#L40 is not adjusted

ad 2:
I think we should adjust ContentTypeContext so that it sets the ItemsPerPage and Parameters properties only when they are given (they should not be required)

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...ldTypeData\/ObjectRelationDataProvider.php":1069176595,"src\/lib\/API\/ContentData\/FieldTypeData\/MediaDataProvider.php":713000072,"src\/lib\/API\/ContentData\/FieldTypeData\/RichTextDataProvider.php":4149789036,"src\/lib\/API\/ContentData\/FieldTypeData\/AuthorDataProvider.php":2116185812,"src\/lib\/API\/ContentData\/FieldTypeData\/DateTimeDataProvider.php":139149081,"src\/lib\/API\/ContentData\/FieldTypeData\/DateDataProvider.php":108427505,"src\/lib\/API\/ContentData\/FieldTypeData\/IntegerDataProvider.php":1210105966,"src\/lib\/API\/ContentData\/FieldTypeData\/TextLineDataProvider.php":1211357950,"src\/lib\/API\/ContentData\/FieldTypeData\/BooleanDataProvider.php":635749480,"src\/lib\/Subscriber\/Review.php":26452211,"src\/lib\/Subscriber\/InitialStage.php":764061016,"src\/lib\/Subscriber\/AbstractProcessStage.php":2581306890,"src\/lib\/Subscriber\/EditContent.php":1686670039,"src\/lib\/Subscriber\/PublishDraft.php":582029031,"src\/lib\/Subscriber\/PublishInTheFuture.php":3673154143,"src\/lib\/Subscriber\/CreateContentDraft.php":2412000011,"src\/lib\/Event\/TransitionEvent.php":1911713306,"src\/lib\/Event\/Events.php":2807281169,"src\/lib\/Event\/InitialEvent.php":2368022602}}
```
Comment (mnocon) [inline .php_cs.cache:1]: This file should not be added

File: src/bundle/Resources/config/services.yaml [hunk 2]
```diff
@@ -70,3 +70,7 @@ services:
             $siteaccess: '@ezpublish.siteaccess'
         tags:
             - { name: console.command }
+
+    EzSystems\Behat\QueryType\FoldersUnderMediaQueryType: ~
+
+
```
Comment (mnocon) [inline src/bundle/Resources/config/services.yaml:76]: Unwanted empty lines

File: src/lib/QueryType/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+### Folders under media query field
```
Comment (mnocon) [inline src/lib/QueryType/README.md:1]: This file should not be added as well

**Review by mnocon — COMMENTED** [review]
Couple of comments before the last commit is here.

**Review by mnocon — APPROVED** [review]
🎉

**Review by bogusez — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Nicely done!

**Pull request merged**